sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormTrainingClient/FormTrainingClientTests.cs

```diff
@@ -99,6 +105,33 @@ public void StartTrainingArgumentValidation()
             Assert.ThrowsAsync<ArgumentNullException>(() => client.StartTrainingAsync((Uri)null));
         }
 
+        [Test]
```

Comment:
following the patterns I've seen in another libraries, it will be good to move the mock tests into their own class.
- [TA](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientMockTests.cs)
- [AppConfig](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationMockTests.cs)

Reply:
Personally I see no reason for treating mock tests differently since they are just non-live tests as well. I don't feel strongly about it, so I'll follow the pattern from other libraries. We can easily revert it back if we think it's not the right choice in the future.